src/TrainerAnnotator.cpp

```diff
@@ -0,0 +1,92 @@
+// Developed by: Rakib
```

Comment:
I would make this into an executable instead of an annotator. The same way extractFeatures is an executable. Since there is no processing going on here, conceptually it does not make sense to have it as an annotator.

Reply:
Ok. I will change that.